src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -162,41 +162,31 @@ private void givenCDAWithCustomCertificateAuthority(URI privateKeyUri, URI certi
         kernel.getContext().waitForPublishQueueToClear();
     }
 
+    private GetCertificateRequest buildCertificateUpdateRequest(
+            GetCertificateRequestOptions.CertificateType type, Consumer<CertificateUpdateEvent> cb) {
+        GetCertificateRequestOptions requestOptions = new GetCertificateRequestOptions();
+        requestOptions.setCertificateType(type);
+        return new GetCertificateRequest("com.aws.clients.Plugin", requestOptions, cb);
+    }
 
     /**
-     * Simulates the request client components make to create their certificates. Clients pass their certificate
-     * chain pem to get verified.
+     * Simulates the subscription client or server component create to get their certificates.
      */
-    private X509Certificate[] arrangeClientComponentCertificateChain() throws CertificateGenerationException,
```

Comment:
This biggerish change you see here is just extracting a few things from here so that we can better simulate an external component subscribing to CDA CA changes through the API